sample/parcel/Sample.jsx

```diff
@@ -4,8 +4,11 @@ import TimePicker from 'react-time-picker';
 import './Sample.less';
 
 export default class Sample extends Component {
-  state = {
-    value: new Date(),
+  constructor(props) {
```

Comment:
Why the change in constructors?

Reply:
Ok, thanks! I'll change the state initialization back and update the PR.